src/api.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+export const TASKS_API_ROOT = '/api/tasks/v1';
```

Comment:
Also talked with @Monnte about why tasks api is used, I know we do not have our backend but I though we will just do 1:1 copy of inventory table.. meaning use same api as they do (https://prod.foo.redhat.com:1337/api/inventory/v1/hosts?per_page=50&page=1 ...) and if we are in need of any additional filtering we could just decide to 'not render' systems that are not usable for ugprades page (e.g. 9.Y systems)

@Wecros WDYT? so far we agreed with @Monnte that he will change it for 1:1 copy with inventory api

Reply:
Already changed to inventory api :+1:

Reply:
agree with the change